bin/cylc-print

```diff
@@ -89,8 +90,8 @@ def get_option_parser():
 
 @cli_function(get_option_parser)
 def main(parser, options, regfilter=None):
-    suite_src_files_mgr = SuiteSrvFilesManager()
-    allsuites = suite_src_files_mgr.list_suites(regfilter)
+    allsuites = get_scan_items_from_fs(reg_pattern=regfilter,
```

Comment:
Will have to wrap the generator in a `list()`.

`allsuites = list(get_scan_items_from_fs(reg_pattern=regfilter, active_only=False))`

Reply:
That I will.